docusaurus.config.en.js

```diff
@@ -326,14 +332,6 @@ const config = {
             label: "English",
             to: "/",
           },
-          {
```

Comment:
Link checker will fail if these are here as we no longer copy across the ru and zh folders